revent/urls.py

```diff
@@ -1,11 +1,11 @@
 from django.contrib import admin
-from django.urls import include, path
+from django.urls import path
+from django.views.decorators.csrf import csrf_exempt
+from strawberry.django.views import GraphQLView
 
-from photo import views
+from photo.schema import schema
 
 urlpatterns = [
     path("admin/", admin.site.urls),
-    path("", views.hello, name="hello")
-    # In case we wanna have multiple url files for each app
-    # path('photo/', include('photo.urls'))
+    path("graphql/", csrf_exempt(GraphQLView.as_view(schema=schema))),
```

Comment:
please go back to using `AsyncGraphQLView`

Reply:
Theres is currently an error with the queries when using AsyncGraphQLView